lib/logstash/plugin.rb

```diff
@@ -121,9 +121,22 @@ def inspect
     end
   end
 
+  public
+  def self.auto_download?
+    @auto_download
```

Comment:
this should be automatically provided by Clamp:

```ruby
require "clamp"
class X < Clamp::Command
  option "--[no-]foo", :flag, "Whatever", :default => false
  def execute
    # if you use a flag, then 'foo?' becomes a method :)
    puts foo?
  end
end

X.run
```

% ruby test.rb --foo
true
% ruby test.rb
false
% ruby test.rb --no-foo
false
```